src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -1,18 +1,160 @@
 package frc.robot.subsystems.intake;
 
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkPIDController;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.constants.RobotConfig;
+import frc.robot.constants.RobotConstants;
+import frc.robot.constants.RobotConstants.IntakeConfig;
 
 public class Intake extends SubsystemBase {
-  /** Creates a new ExampleSubsystem. */
-  public Intake() {}
+  private final CANSparkMax m_rollerFollowerMotor; // left motor (Rollers)
+  private final CANSparkMax m_rollerLeaderMotor; // right motor (Rollers)
+
+  private final SparkPIDController m_rollerPidController; // Roller velocity PID controllers
+  private final RelativeEncoder m_rollerEncoder; // Roller Relative Encoder
+
+  private final DigitalInput m_intakeSensor; // Line break note sensor
+
+  private boolean m_testModeCheck1; // Booleans for if test mode is enabled
```

Comment:
remove this, we don't need PID constants checks (or PID at all)

Reply:
Yep.

Reply:
Removed alongside PID in "Removed PID commit"